src/Platform/Hosted/Helpers/ResourceTokenUtils.ts

```diff
@@ -7,11 +7,11 @@ export interface ParsedResourceTokenConnectionString {
 }
 
 export function parseResourceTokenConnectionString(connectionString: string): ParsedResourceTokenConnectionString {
-  let accountEndpoint: string;
-  let collectionId: string;
-  let databaseId: string;
-  let partitionKey: string;
-  let resourceToken: string;
+  let accountEndpoint = "";
+  let collectionId = "";
+  let databaseId = "";
+  let partitionKey = "";
+  let resourceToken = "";
```

Comment:
Undo this. Make the type allow undefined

Reply:
Actually we need to initiate variable. 
`Variable  is used before being assigned`